sycl/plugins/cuda/pi_cuda.hpp

```diff
@@ -395,7 +396,9 @@ struct _pi_queue {
     cuda_piDeviceRelease(device_);
   }
 
-  native_type get() const noexcept { return stream_; };
+  native_type get() const noexcept { return stream_; }; // TODO what about memStream_?
+
+  native_type get_mem() const noexcept { return memStream_; };
```

Comment:
I would call this "get_alternative_stream()" or similar, rather than "mem", because mem is a bit confusing and, from the perspective of the PI queue class, what operations go to each stream is irrelevant.